modules/services/karabiner-elements/default.nix

```diff
@@ -91,26 +102,35 @@ in
+        rm -rf /run/wrappers
+        mkdir -p /run/wrappers/bin
+        install -m4555 "${package}/Library/Application Support/org.pqrs/Karabiner-Elements/bin/karabiner_session_monitor" /run/wrappers/bin
       '';
       serviceConfig.RunAtLoad = true;
       serviceConfig.KeepAlive.SuccessfulExit = false;
     };
-
     launchd.user.agents.karabiner_session_monitor = {
       serviceConfig.ProgramArguments = [
-        "/bin/sh" "-c"
+        "/bin/sh"
+        "-c"
         "/bin/wait4path /run/wrappers/bin && /run/wrappers/bin/karabiner_session_monitor"
       ];
-      serviceConfig.Label = "org.pqrs.karabiner.karabiner_session_monitor";
+      serviceConfig.Label = "org.pqrs.service.agent.karabiner_session_monitor";
       serviceConfig.KeepAlive = true;
       managedBy = "services.karabiner-elements.enable";
     };
 
-    environment.userLaunchAgents."org.pqrs.karabiner.agent.karabiner_grabber.plist".source = "${cfg.package}/Library/LaunchAgents/org.pqrs.karabiner.agent.karabiner_grabber.plist";
-    environment.userLaunchAgents."org.pqrs.karabiner.agent.karabiner_observer.plist".source = "${cfg.package}/Library/LaunchAgents/org.pqrs.karabiner.agent.karabiner_observer.plist";
-    environment.userLaunchAgents."org.pqrs.karabiner.karabiner_console_user_server.plist".source = "${cfg.package}/Library/LaunchAgents/org.pqrs.karabiner.karabiner_console_user_server.plist";
+    # from launch agents folder in karabiner elements executable
+    environment.userLaunchAgents."org.pqrs.service.agent.karabiner_grabber.plist".source =
+      "/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Non-Privileged Agents.app/Contents/Library/LaunchAgents/org.pqrs.service.agent.karabiner_grabber.plist";
+    environment.userLaunchAgents."org.pqrs.service.agent.karabiner_console_user_server.plist".source =
+      "/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Non-Privileged Agents.app/Contents/Library/LaunchAgents/org.pqrs.service.agent.karabiner_console_user_server.plist";
+    environment.userLaunchAgents."org.pqrs.service.agent.Karabiner-Menu.plist".source =
+      "/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Non-Privileged Agents.app/Contents/Library/LaunchAgents/org.pqrs.service.agent.Karabiner-Menu.plist";
+    environment.userLaunchAgents."org.pqrs.service.agent.Karabiner-MultitouchExtension.plist".source =
+      "/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Non-Privileged Agents.app/Contents/Library/LaunchAgents/org.pqrs.service.agent.Karabiner-MultitouchExtension.plist";
+    environment.userLaunchAgents."org.pqrs.service.agent.Karabiner-NotificationWindow.plist".source =
+      "/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Non-Privileged Agents.app/Contents/Library/LaunchAgents/org.pqrs.service.agent.Karabiner-NotificationWindow.plist";
+
```

Comment:
I think these should come from inside the package instead and shouldn't require linking `/Library/Application Support`

We already do some level of patching in Nixpkgs to try to make these work https://github.com/NixOS/nixpkgs/commit/cfe3c23ad5c61c6151ccb0879a467391121f0553